ResourceGenerator/readme.template.md

```diff
@@ -0,0 +1,6 @@
+
+# <ResourceFriendlyName>
+
+## Description
+
+This resource configures an <ResourceDescription> Device.
```

Comment:
I would remove "device" from the template. This might be confusing, if we generate other resources with the DRG.

Reply:
Sure will do in the evening